diffsims/tests/test_utils/test_sim_utils.py

```diff
@@ -132,3 +145,176 @@ def test_kinematic_simulator_invalid_illumination():
 def test_uvtw_to_uvw(uvtw, uvw):
     val = uvtw_to_uvw(uvtw)
     np.testing.assert_almost_equal(val, uvw)
+
+
+class TestHolzCalibration:
+    def test_get_holz_angle(self):
+        wavelength = 2.51 / 1000
+        lattice_parameter = 0.3905 * 2 ** 0.5
+        angle = get_holz_angle(wavelength, lattice_parameter)
+        assert approx(95.37805 / 1000) == angle
```

Comment:
(see comment above) ~ something like this reads weirdly, floating point numbers are never well described as `==` to one another

Reply:
ie) I would use `np.isclose(angle,96.37805/1000)`